doc/source/reference/main_namespace.rst

```diff
@@ -0,0 +1,49 @@
+========================
+The main SciPy namespace
+========================
+
+.. currentmodule:: scipy
+
+The main ``scipy`` namespace has very few objects in it by design. Only show
+generical functionality related to testing, build info and versioning, and one
+class (`LowLevelCallable`) that didn't fit in one of the submodules, are
+present:
+
+.. autosummary::
+   :toctree: generated/
+
+   LowLevelCallable
+   show_config
+   test
+
+
+The one public attribute is:
+
+================== ===============================================
+``__version__``    SciPy version string
+================== ===============================================
+
+
+Submodules
+----------
+
+=============  ==================================================
+`cluster`      Clustering functionality
+`constants`    Physical and mathematical constants and units
+`datasets`     Load SciPy datasets
+`fft`          Discrete Fourier and related transforms
+`fftpack`      Discrete Fourier transforms (legacy)
+`integrate`    Numerical integration
+`interpolate`  Interpolation
+`io`           Scientific data format reading and writing
+`linalg`       Linear algebra functionality
+`misc`         Utility routines (deprecated)
```

Comment:
I am not sure about keeping legacy and deprecated things. Legacy could make sense, but deprecated a bit less to me.

Reply:
I'd like to reject this comment @tupui, and keep things as they are. We have it in the API reference and in the sidebar. Once the deprecation expires it can be cleaned up. This is standard for all deprecated functionality - no reason to deviate here.